Source/Scene/Cesium3DTileset.js

```diff
@@ -0,0 +1,1821 @@
+         */
+        this.debugWireframe = defaultValue(options.debugWireframe, false);
+
+        /**
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, renders the bounding volume for each visible tile.  The bounding volume is
+         * white if the tile has a content bounding volume; otherwise, it is red.  Tiles that don't meet the
+         * screen space error and are still refining to their descendants are yellow.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.debugShowBoundingVolume = defaultValue(options.debugShowBoundingVolume, false);
+
+        /**
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, renders the bounding volume for each visible tile's content. The bounding volume is
+         * blue if the tile has a content bounding volume; otherwise it is red.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.debugShowContentBoundingVolume = defaultValue(options.debugShowContentBoundingVolume, false);
+
+        /**
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, renders the viewer request volume for each tile.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.debugShowViewerRequestVolume = defaultValue(options.debugShowViewerRequestVolume, false);
+
+        /**
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, draws labels to indicate the geometric error of each tile.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.debugShowGeometricError = defaultValue(options.debugShowGeometricError, false);
+
+        /**
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, draws labels to indicate the number of commands, points, triangles and features of each tile.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.debugShowRenderingStatistics = defaultValue(options.debugShowRenderingStatistics, false);
+
+        /**
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, draws labels to indicate the geometry and texture memory usage of each tile.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.debugShowMemoryUsage = defaultValue(options.debugShowMemoryUsage, false);
+
+        var that = this;
+
+        // We don't know the distance of the tileset until tileset.json is loaded, so use the default distance for now
```

Comment:
I assume when we merge in the new request schedule, we won't throttle this initial call at all, right?  Seems like tileset.json requests should be done ASAP.